### PR TITLE
fixed 'Getting Started' in readme

### PR DESCRIPTION
**Pull request by nnisarggada**
- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/dhruvdabhi101/qUIck-React/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/dhruvdabhi101/qUIck-React/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] My changes generate no new warnings.
- [X] I have added all README.md files which are specified in the [CONTRIBUTING GUIDELINE](https://github.com/dhruvdabhi101/qUIck-React/blob/main/.github/CONTRIBUTING_GUIDELINE.md)'s file structures section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 2. **Clone your Repository:** Clone your forked repository URL to your local machine using this command:
 
     ```
-    git clone <your-cloned-repo-url>
+    git clone https://github.com/developer-student-clubs/qUIck-react/
```
Comment (dhruvdabhi101) [inline README.md:21]: No, You have to put your **cloned** repo URL there.

Comment (nnisarggada) [reply README.md:21]: I cannot do `Step 3 - cd qUIck-react` mentioned in the README.md without cloning this repo first. I think and extensive solution would be to clone this repo and the copy the content of src/components folder to the user's repo's src folder.

**Comment by dhruvdabhi101** [comment]
I don't see any major change in this Pull request. Check other issues please. : )
Thanks for contributing.

File: README.md (hunk 1 above)
Comment (nnisarggada) [reply README.md:21]: I am sorry. I had misunderstood.

**Pull request closed without merging**